components/suggestion/switch_channel_provider.jsx

```diff
@@ -64,6 +67,11 @@ class SwitchChannelSuggestion extends Suggestion {
 
         const member = this.props.channelMember;
         let badge = null;
+
+        const state = getState();
```

Comment:
We don't allow calling global state unless its in a connector, If I remember correctly I connected some of the providers when refactoring the use of global state in our utils functions. You can check some of them for reference

Reply:
@asaadmahmood ^